### PR TITLE
Map size color config

### PR DESCRIPTION
**Pull request by llmlks**

### CODE REVIEW

**Comment by llmlks** [comment]
Found a bug in the implementation, no need for code review yet

**Comment by llmlks** [comment]
Bug is now fixed

**Comment by henrhaat** [comment]
Other than the Otaniemi catalog displaying in the map incorrectly, which we talked about, I didn't find any issues.

**Comment by llmlks** [comment]
Still waiting for some improvements in the normalization of size column values

**Comment by llmlks** [comment]
Normalization of the size column is now computed on the whole uploaded data set, so the sizes are comparable across time frames.

File: components/quake_map.py [hunk 1]
```diff
@@ -16,13 +17,17 @@
 OpenStreetMap contributors</a>"""
 
 
-def get_component(eq_data):
+def get_component(eq_data, sizes, color_column=None):
```
Comment (henrhaat) [inline components/quake_map.py:20]: 'sizes' is referred as 'size_column' in the docstring.

File: utils/earthquake_data.py [hunk 2]
```diff
@@ -84,6 +85,33 @@ def filter_by_dates(self, datemin, datemax):
...
+    def get_normalized_column(self, column_name):
+        """Return a numpy array containing the values from the
+        given column normalized to the range [0,1].
+
+        If the column does not exist, a None value is returned.
+        No normalization is performed on columns that are not
+        numeric.
+
+        Keyword arguments:
+        column_name -- Column to normalise
```
Comment (henrhaat) [inline utils/earthquake_data.py]: You use both 'normalise' and 'normalize' here.

**Comment by henrhaat** [comment]
The configs works with the float value -columns. All the columns from the Otaniemi data were not as choices in the drop-down menus (the ones with ',' in the values), which is probably related to the same issue as with the locations.

**Comment by henrhaat** [comment]
This is ready to merge

**Pull request merged**